ruma-events/src/enums.rs

```diff
@@ -146,13 +151,93 @@ pub enum AnyRoomEvent {
 }
 
 /// Any room event stub (room event without a `room_id`, as returned in `/sync` responses)
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 #[serde(untagged)]
 pub enum AnyRoomEventStub {
     /// Any message event stub
     Message(AnyMessageEventStub),
     /// `"m.room.redaction"` stub
     Redaction(RedactionEventStub),
     /// Any state event stub
-    StateEvent(AnyStateEventStub),
+    State(AnyStateEventStub),
```

Comment:
Whoops :D

Reply:
I see matrix json objects and ruma type names sometimes when I close my eyes so it's totally understandably :laughing: !!